tests/gdb-tests/tests/test_command_plist.py

```diff
@@ -9,14 +9,74 @@
 LINKED_LISTS_BINARY = tests.binaries.get("linked-lists.out")
 
 
-def startup(start_binary):
+def startup(start_binary) -> None:
     start_binary(LINKED_LISTS_BINARY)
 
     gdb.execute("break break_here")
     gdb.execute("run")
     gdb.execute("up")
 
 
+def test_command_plist_dereference_limit_change_has_impact_on_plist(start_binary):
+    """
+    Tests the plist command with different dereference limits
+    """
+    startup(start_binary)
+    gdb.execute("set dereference-limit 5")
+    expected_out = re.compile(
+        """\
+0[xX][0-9a-fA-F]+ <node_a>: {\\s*
```

Comment:
I wonder if this would be more readable if we formatted it with `f"..."` and do sth like `{identifier} {whitespaces}` etc.

But this is good enough anyway.

Reply:
I was thinking the same, but ATM no time to implement this.